libs/sdk-ui-dashboard/src/presentation/topBar/buttonBar/types.ts

```diff
@@ -13,6 +13,7 @@ import {
  */
 export interface IButtonBarProps {
     buttons?: React.ReactNode;
+    childContentPosition?: "left" | "right";
```

Comment:
Describe this props

Reply:
Define default "left"

Reply:
I added the note for this new prop and the default value in the DefaultButtonBar component. Thanks!